nservicebus/upgrades/5to6-migrate-existing-handlers.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Migrate existing handlers/sagas to Version 6
+summary: Describes how to migrate existing handlers/sagas to Version 6
+---
+
+In previous versions of NServiceBus, a typical handler class looks like the below:
```

Comment:
note this deals with property injection when most of our doco uses constructor injection. how do we clarify this?